cms/models.py

```diff
@@ -447,3 +449,15 @@ def save(self, *args, **kwargs):
             self.slug = "%s-%d" % (original_slug, x)
 
         super().save(*args, **kwargs)
+
+
+@register_snippet
+class SiteNotification(models.Model):
+    """ Snippet model for showing site notifications. """
+
+    message = RichTextField(max_length=255)
```

Comment:
We should limit the available controls so that images and other non-text content don't end up in here that could cause layout issues. I think this is probably a reasonable set:

```suggestion
    message = RichTextField(max_length=255, features=[ 'bold', 'italic', 'link', 'document-link'])
```